tests/storage/blockvolume_test.py

```diff
@@ -112,17 +112,20 @@ def test_fail_invalid_block_volume_size(self, preallocate, vol_format):
 class TestBlockVolumeManifest(VdsmTestCase):
 
     @contextmanager
-    def make_volume(self, size, storage_type='block', format=sc.RAW_FORMAT):
+    def make_volume(self, size, format=sc.RAW_FORMAT, prealloc=sc.SPARSE_VOL):
+        storage_type = 'block'
```

Comment:
Good change - but since we really used this once, we can simply inline "block" in the single line using this.